test/cov.jl

```diff
@@ -88,62 +88,62 @@ end
 # weighted covariance
 
 if VERSION < v"0.5.0-dev+679"
-    @test cov(X, wv1)           ≈ S1w ./ sum(wv1)
-    @test cov(X, wv2; vardim=2) ≈ S2w ./ sum(wv2)
+    @test cov(X, wv1; corrected=false)           ≈ S1w ./ sum(wv1)
```

Comment:
The `corrected` argument wouldn't be supported at all for `WeightVec`, or at least and error would be raised if `corrected=true` is passed.